src/NzbDrone.Core/Datastore/Migration/142_add_priority_to_indexers.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+using NzbDrone.Core.Indexers;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(142)]
```

Comment:
This conflicts with the migration in import lists: https://github.com/Sonarr/Sonarr/pull/3398

Please change to 143.